src/scenes/CreditsScene.ts

```diff
@@ -111,112 +112,113 @@ export class CreditsScene extends Scene<FriendlyFire> {
+        CreditsScene.headlineFont.drawText(ctx, titleText, position, 'white');
 
         CreditsScene.standardFont.drawText(
             ctx,
             versionText,
-            posX, posY + this.headlineCharHeight + gap,
+            position.moveYBy(this.headlineCharHeight + gap),
             'white'
         );
 
-        return posY + this.headlineCharHeight + this.standardCharHeight + gap + 20
+        return position.moveYBy(this.headlineCharHeight + this.standardCharHeight + gap + 20);
     }
 
     private drawParagraph(
-        ctx: CanvasRenderingContext2D, posY: number, posX: number, lines: string[], marginBotton = 10
-    ): number {
-        let y = posY;
-
+        ctx: CanvasRenderingContext2D, position: Point, lines: string[], marginBotton = 10
+    ): Point {
         lines.forEach(line => {
-            CreditsScene.standardFont.drawText(ctx, line, posX, y, "white");
-            y += this.standardCharHeight;
-        });
+            CreditsScene.standardFont.drawText(ctx, line, position, 'white');
+            position.moveYBy(this.standardCharHeight);
+        })
 
-        return y + marginBotton;
+        return position.moveYBy(marginBotton);
     }
 
     private drawCredit(
-        ctx: CanvasRenderingContext2D, posY: number, posX: number, title: string, names: string[]
-    ): number {
-        let y = posY;
+        ctx: CanvasRenderingContext2D, position: Point, title: string, names: string[]
+    ): Point {
         const gap = 5;
-        CreditsScene.creditsFont.drawText(ctx, title, posX, y, "white");
-        y += this.creditsFontHeight + this.lineSpacing + gap;
+
+        CreditsScene.creditsFont.drawText(ctx, title, position, 'white');
+        position.moveYBy(this.creditsFontHeight + this.lineSpacing + gap);
 
         names.forEach(name => {
-            CreditsScene.standardFont.drawText(ctx, name, posX, y, "white");
-            y += this.standardCharHeight;
+            CreditsScene.standardFont.drawText(ctx, name, position, 'white');
+            position.moveYBy(this.standardCharHeight);
         })
-        return y + 40;
-    }
 
+        return position.moveYBy(40);
+    }
 
-    public draw(ctx: CanvasRenderingContext2D, width: number, height: number) {
+    public draw(ctx: CanvasRenderingContext2D) {
         ctx.save();
         ctx.drawImage(CreditsScene.backgroundImage, 0, 0);
 
         // Stars
-        this.starPositions.forEach((pos, index) => {
+        this.starPositions.forEach((position, index) => {
             const starIndex = index % CreditsScene.stars.length;
-            CreditsScene.stars[starIndex].drawTag(ctx, "idle", pos[0], pos[1], this.time * 1000);
+            CreditsScene.stars[starIndex].drawTag(ctx, 'idle', position, this.time * 1000);
         });
 
         // Leaf
-        CreditsScene.leaf.drawTag(ctx, "idle", 414, 163, this.time * 1000);
+        CreditsScene.leaf.drawTag(ctx, "idle", new Point(414, 163), this.time * 1000);
```

Comment:
Two new point objects per frame in this method. Well, it's the credits scene, so this is not important for performance by I mention it anyway.